common/src/main/java/com/wynntils/services/mapdata/MapDataService.java

```diff
@@ -259,6 +262,34 @@ public float calculateVisibility(ResolvedMapVisibility mapVisibility, float zoom
         return 0;
     }
 
+    public double getMarkerVisibility(Location location, ResolvedMarkerOptions resolvedMarkerOptions) {
```

Comment:
This was the placement I was looking for, thanks! :) To tie in better with the function above, maybe rename it `calculateMarkerVisibility`? It's not really a trivial getter which the name currently kind of implies.